### PR TITLE
Update references to Weave Cloud

### PR DESCRIPTION
**Pull request by pidster**
Adds a message to `weave` script and updates the README to make it a bit more sane.

### CODE REVIEW

File: weave [hunk 1]
```diff
@@ -95,6 +95,8 @@ where <peer>     = <ip_address_or_fqdn>[:<port>]
       <endpoint> = [tcp://][<ip_address>]:<port> | [unix://]/path/to/socket
       <peer_id>  = <nickname> | <weave internal peer ID>
       <mode>     = consensus[=<count>] | seed=<mac>,... | observer
+
+To troubleshoot and visualize Weave networks, try Weave Cloud: http://www.weave.works/product/cloud/
```
Comment (bboreham) [inline weave]: How about "To troubleshoot and visualize your cluster" - it does much more than just the Weave network part.

Comment (pidster) [inline weave]: I was trying to make it relevant to Net

Comment (pidster) [inline weave]: NB My laptop is b0rked so I won't be able to make any edits in the foreseeable.

Comment (bboreham) [inline weave]: I think as "a word from our sponsor" it's fine to blab about vaguely-related things.
So would we explictly say something like "Weave Net is brought to you by the makers of ..." ?

**Pull request merged**